src/components/Answer/index.tsx

```diff
@@ -7,30 +9,71 @@ import MultiInputs from 'components/MultiInputs';
 import Nps from 'components/Nps';
 import Rating from 'components/Rating';
 import TextArea from 'components/TextArea';
-import { Answer as AnswerType } from 'types/answer';
+import { Answer as AnswerType, instanceOfAnswer } from 'types/answer';
 import { DisplayType, Question, getDisplayTypeEnum } from 'types/question';
-import { AnswerRequest } from 'types/request/surveySubmitRequest';
+import { AnswerRequest, instanceOfAnswerRequest } from 'types/request/surveySubmitRequest';
 
 export const answerDataTestIds = {
   base: 'answer__base',
 };
 
 interface AnswerProps {
   question: Question;
+  onAnswerChanged: (answers: AnswerRequest[]) => void;
```

Comment:
Why don't we just set it as an optional field so we don't need to add it every time we need it?

Reply:
Currently, this component requires a callback when called in one place. In the future, if it's called in other places that don't need the callback, I will make it optional.😄 Please ignore this 🙏